test/e2e/storage/drivers/in_tree.go

```diff
@@ -1503,31 +1505,24 @@ func (a *azureDriver) GetDynamicProvisionStorageClass(config *testsuites.PerTest
 	return testsuites.GetStorageClass(provisioner, parameters, nil, ns, suffix)
 }
 
-func (a *azureDriver) PrepareTest(f *framework.Framework) (*testsuites.PerTestConfig, func()) {
+func (a *azureDiskDriver) PrepareTest(f *framework.Framework) (*testsuites.PerTestConfig, func()) {
 	return &testsuites.PerTestConfig{
 		Driver:    a,
 		Prefix:    "azure",
 		Framework: f,
 	}, func() {}
 }
 
-func (a *azureDriver) CreateVolume(config *testsuites.PerTestConfig, volType testpatterns.TestVolType) testsuites.TestVolume {
+func (a *azureDiskDriver) CreateVolume(config *testsuites.PerTestConfig, volType testpatterns.TestVolType) testsuites.TestVolume {
 	ginkgo.By("creating a test azure disk volume")
-	if volType == testpatterns.InlineVolume {
```

Comment:
this is removed because this testpattern is not supported?

Reply:
This if-statement was causing some pod scheduling error. We are trying to run the e2e test suites in an Azure cluster so we don't have `CloudConfig.Zone` (extracted from `--gce-zone`) populated.